app/models.py

```diff
@@ -26,6 +26,8 @@ class Donor(models.Model):
     # phone_regex = RegexValidator(regex=r'^\+?1?\d{9,15}$',
     # message = ('Phone number must be entered in the format: '+999999999'. Up
     # to 15 digits allowed.'))
+    created_at = models.DateField(auto_now=True, verbose_name="Date Created")
```

Comment:
These should be DateTimeField

Reply:
Why? Then wouldn't it make 2017-07-10-17:30:00 and 2017-07-10-17:30:01 a different time domain, whereas we want them to be same?